data/.gitignore

```diff
@@ -0,0 +1,15 @@
+#
+# (c) 2023 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+#
+cca_test/
+detray_simulation/
+single_module/
+tml_detector/
+tml_full/
+tml_pixel_barrel/
+tml_pixels/
+two_modules/
+*.tar.gz
+*.md5
```

Comment:
Let's just set this to `*` so it automatically adapts to future additions to the data files.

Reply:
But then it would also ignore the scripts in the directory. :confused:

Reply:
Yeah, no...

Reply:
Well, you can lead someone to water I guess. :smile: